### PR TITLE
Webcam

### PR DESCRIPTION
**Pull request by sandeshsoni**
@GantMan I am adding a few comments with problem description with images. Please you add your thoughts and I can fix accordingly.
I need to reindent the file as my IDE uses some default 4space and I broke the convention.

### CODE REVIEW

File: example/nsfw_demo/src/App.js [hunk 1]
```diff
@@ -138,16 +138,22 @@ class App extends Component {
...
   detectWebcam = async () => {
-    const video = document.querySelectorAll('.captureCam')
-    const predictions = await this.state.model.classify(video[0])
-    let droppedImageStyle = this.detectBlurStatus(predictions[0].className)
-    this.setState({
-      message: `Identified as ${predictions[0].className}`,
-      predictions,
-      droppedImageStyle
-    })
+      // await this.sleep(100)
```
Comment (sandeshsoni) [inline example/nsfw_demo/src/App.js]: @GantMan here is a strange behaviour.

If I dont add sleep 100,
The predictions are not updated.

If I add sleep 100, predictions are updated, But, they update for sometime even after I close the webcam.

screenshot 2019-03-04 22 43 49

Comment (sandeshsoni) [reply example/nsfw_demo/src/App.js]: no update if not made sleep 100

screenshot 2019-03-04 22 43 06

File: example/nsfw_demo/src/App.js [hunk 2]
```diff
@@ -138,16 +138,22 @@ class App extends Component {
...
+
+      const video = document.querySelectorAll('.captureCam')
+
+      if (video[0]){
```
Comment (sandeshsoni) [inline example/nsfw_demo/src/App.js]: if (video[0]){

- value is undefined at the start.... it crashes if taken further

File: example/nsfw_demo/src/App.js (hunk 1 above)
Comment (sandeshsoni) [reply example/nsfw_demo/src/App.js]: predictions aren't updated at all if I don't set it to sleep for 100ms.

**Comment by GantMan** [comment]
This looks great!   Do you want to add anything else?  If not, I can merge and then put some finishing touches.

**Comment by sandeshsoni** [comment]
@GantMan I dont have anything to add. You may proceed.

I left the console  log messages for now. Thank you

**Pull request merged**